pkg/cmd/cli/describe/describer.go

```diff
@@ -418,6 +420,36 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 	})
 }
 
+// OAuthAccessTokenDescriber generates information about an OAuth Acess Token (OAuth)
+type OAuthAccessTokenDescriber struct {
+	client.Interface
+}
+////--
+func (d *OAuthAccessTokenDescriber) Describe(namespace, name string) (string, error) {
+	c := d.OAuthAccessTokens()
+	oAuthAccessToken, err := c.Get(name)
+	if err != nil {
+		return "", err
+	}
+
+	return tabbedString(func(out *tabwriter.Writer) error {
+		formatMeta(out, oAuthAccessToken.ObjectMeta)
+		formatString(out, "Client Name", oAuthAccessToken.ClientName)
+		formatString(out, "Expires In", oAuthAccessToken.ExpiresIn)
+		formatString(out, "Redirect URI", oAuthAccessToken.RedirectURI)
```

Comment:
i don't think this value is very useful.  The redirect already happened and someone doing a post-mortem on a token will have the json.